go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	github.com/huandu/xstrings v1.2.0 // indirect
 	github.com/imdario/mergo v0.3.9
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v0.0.0-20210409143810-a99ffa1cac67
+	github.com/openshift/api v0.0.0-20210629145910-15a1cae1fca8
```

Comment:
@Elbehery please squash your commits and add a descriptive message stating why we're bumping `openshift/api`. You'll be able to re-use it in the other PRs.

The PR description is a good start :wink:

Reply:
@bertinatto done 👍🏽  🙏🏽

Reply:
@Elbehery sorry for picking on this, but the commit message isn't quite correct: we are not *enabling* the feature gates here, the user is the one who enables them (through the`TechPreviewNoUpgrade` featureSet).

I meant the commit message could be something like this (for this PR specifically):

```
Bump openshift/api

The goal of this bump is to apply `CSIMigrationGCE` and `CSIMigrationAzureDisk`
feature gates to kubelet config when the `TechPreviewNoUpgrade` featureSet is
set by the user.

```

Reply:
when you get a chance could you update ^^ @Elbehery